### ISSUE TITLE
Mod Icons Wrong size

### ISSUE THREAD

**Issue by Rdna123** [gorilla-devs/GDLauncher#883]
**Describe the bug**
The Icons for mods from curseforge browsing scaled wrong. 

**To Reproduce**
Go to 'Fabric Instance → manage → add mod'

**Expected behavior**
Full image shown in icon box

**Possible solutions**
Rescale function in icon loader.

**Screenshots**
GDLauncher
![image](https://user-images.githubusercontent.com/54755407/116773192-ded6a000-aa08-11eb-9e18-184272f626ca.png)

CurseForge
![image](https://user-images.githubusercontent.com/54755407/116773226-104f6b80-aa09-11eb-9d45-8cca052cc446.png)

**Operating System:**
 - OS: Windows
 - Version: 10

**Comment by RemagOfficial** [comment]
fixed in 1.1.5

**Comment by Rdna123** [comment]
I have 1.1.5 unless I had an update error I will try reinstalling it to see
if it works then.

**Comment by benjiae** [comment]
it worked @Rdna123 ?? asking because it happened to me too

**Comment by Rdna123** [comment]
I am currently unable to check till later to day or tomorrow currently on a
weekend trip.

**Comment by Rdna123** [comment]
It is still there after an uninstall and reinstall 
![image](https://user-images.githubusercontent.com/54755407/116844217-4f132c00-ab97-11eb-95c8-fc2d44746850.png)
![image](https://user-images.githubusercontent.com/54755407/116844163-24c16e80-ab97-11eb-9ae8-04b9ff79ffa8.png)

**Comment by Mrbysco** [comment]
Still here after update 1.1.7
![image](https://user-images.githubusercontent.com/9308260/119178094-89bd0580-ba6d-11eb-9608-200fab358ae4.png)
![image](https://user-images.githubusercontent.com/9308260/119178113-92154080-ba6d-11eb-9db0-1445ee047c69.png)

**Comment by Dqu1J** [comment]
I was able to fix that by adding `!important` in the `background-size: cover` in `.bsywew` css class.

**Comment by Azarattum** [comment]
1.1.14 the issue persists.
![image](https://user-images.githubusercontent.com/43073346/130565299-d597f590-9abf-4e02-a603-b73d18eafe0f.png)
![image](https://user-images.githubusercontent.com/43073346/130565369-539ff154-d490-4e94-b62c-2d0e28219af1.png)
It looks like `background-size` is overridden by something:
![image](https://user-images.githubusercontent.com/43073346/130565677-8f478e37-0395-4026-8390-0f815fadf507.png)

**Comment by Eskaan** [comment]
Yes. I found the issue a few days ago and fixed it for you in 5a245c99cb77d9e5d6cf69c1ece90de6d435d640. Upcoming release (1.1.15

**Comment by Eskaan** [comment]
> 1.1.14 the issue persists.
> ![image](https://user-images.githubusercontent.com/43073346/130565299-d597f590-9abf-4e02-a603-b73d18eafe0f.png)
> ![image](https://user-images.githubusercontent.com/43073346/130565369-539ff154-d490-4e94-b62c-2d0e28219af1.png)
> It looks like `background-size` is overridden by something:
> ![image](https://user-images.githubusercontent.com/43073346/130565677-8f478e37-0395-4026-8390-0f815fadf507.png)

Also, yes! Good job. It was overridden by the „center center“
